core/bin/external_node/src/config/mod.rs

```diff
@@ -382,6 +382,9 @@ pub(crate) struct OptionalENConfig {
     /// Timeout to wait for the Merkle tree database to run compaction on stalled writes.
     #[serde(default = "OptionalENConfig::default_merkle_tree_stalled_writes_timeout_sec")]
     merkle_tree_stalled_writes_timeout_sec: u64,
+    /// Enables the stale keys repair task for the Merkle tree.
+    #[serde(default)]
+    pub merkle_tree_repair_stale_keys: bool,
```

Comment:
I'm not sure whether it's worth starting with ENs or the main node to test the task (bogus stale keys could theoretically be present in either case). EN looks like a less risky option.